### DISCUSSION TITLE
Search function for patterns

### DISCUSSION THREAD

**Discussion by Ged2323** [danielmiessler/Fabric#416] [Ideas]
Hi,
What code was this built with I'd like to improve it by just adding a search engine to look though the patterns or maybe a section of last used patterns.

Regards, 
Gerard Grundy

**Comment by snafu4** [comment]
I suggested a way to search  (https://github.com/danielmiessler/fabric/issues/318) and included my initial script but it was brushed off.

Neil

**Reply by Ged2323** [reply]
Also be good when you're updating pattern it would be good if the original pattern appears when you go to update it.
